docs/README.md

```diff
@@ -2,32 +2,55 @@
- - [FlowForge Cloud](./cloud/) - details specific to the hosted service
- - [Administrator documentation](./admin/) - how to administer the platform
- - [User documentation](./user/) - how to use the platform
+ - [FlowForge Concepts](./user/concepts.md) - Understanding the core terminology and concepts that make up the FlowForge platform.
+ - [User documentation](./user/) - Detailed documentation on how to use FlowForge.
+ - [Migrating from Node-RED to FlowForge](./user/migration/#migrating-a-node-red-application-to-flowforge) - Read our guide on the steps requireed to move your flows from a standalone Node-RED deployment into a production-ready instance of FlowForge.
  - [API Documentation](./api/) - Developer API documentation
- - [Contributing to FlowForge](./contribute/) - how to contribute to the Open Source project
 
-## Installing FlowForge
+## Device Agent (Edge Compute)
+
+- [Install the Device Agent](./user/devices.md) - Install the Device Agent on your own hardware to remotely manage your Node-RED instances.
+- [Registering your Device](./user/devices.md#register-the-device) - Connect with Device with FlowForge.
+- [Deploying flows to your Device](./user/devices.md#deploying-a-node-red-instance-to-the-device) - Learn how to remotely deploy flows to your Device.
+- [Editing flows on your Device](.//user/devices.md#editing-the-node-red-flows-on-a-device) - Setup "Developer Mode" to enable editing of your flows, directly on your Device.
+
+## FlowForge Cloud
+ - [FlowForge Cloud](./cloud/) - find out how we've configured FlowForge for FlowForge Cloud.
+ - [Network Connections](./cloud/#network-connections/) - Details what connections can and cannot be established to Node-RED instances running in FlowForge Cloud.
+ - [Single Sign On](./cloud/#single-sign-on/) - FlowForge supports configuring SAML-based Single Sign-On for particular email domains. Contact us to configure for your team.
+ - [Billing](./cloud/) - find out how we've configured FlowForge for FlowForge Cloud.
+
+## Managing FlowForge
 
 - [Installing FlowForge](./install/) - how to install the platform
-- [Upgrading FlowForge](./upgrade/) - how to upgrade the platform
-- [Install the Device Agent](./user/devices.md) - deploy flows to remote devices
+- [Administrator documentation](./admin/) - how to administer the platform
 
-## Upgrading FlowForge
+## Upgrading & Billing
 
 - [Upgrade your FlowForge Instance](./upgrade/README.md)
+- [Upgrading to Premium](./upgrade/) - how to upgrade the platform
 - [From Open Source to Premium](./upgrade/open-source-to-premium.md)
+
+ ## Contributing to FlowForge
+ - [Useful Information](./contribute/#contributing-to-flowforge/) - Learn the foundational concepts of how FlowForge is built & structured. 
+ - [Development Setup](./contribute/#development-setup/) - Configure your local development environment to contribute to FlowForge.
+ - [Testing](./contribute/#testing/) - Understand our testing philosophy at FlowForge.
```

Comment:
```suggestion
 - [Testing](./contribute/#testing/) - Understand our testing philosophy at FlowForge.
 
```

Reply:
have you changed something here? They look identical?

Reply:
I've added a new line